networkx/linalg/spectrum.py

```diff
@@ -32,11 +32,23 @@ def laplacian_spectrum(G, weight="weight"):
     Notes
     -----
     For MultiGraph/MultiDiGraph, the edges weights are summed.
-    See to_numpy_array for other options.
+    See :func:`~networkx.convert_matrix.to_numpy_array` for other options.
 
     See Also
     --------
     laplacian_matrix
+
+    Examples
+    --------
+    The multiplicity of O as an eigenvalue of the laplacian matrix is equal
+    to the number of connected components of G.
+
+    >>> import numpy as np
+    >>> G = nx.graph_atlas(22) #This graph from the Graph Atlas has 3 connected components.
```

Comment:
IMO it'd make the example more clear if the reader knew what the graph was (I doubt many have the graph atlas memorized :joy: ):

```suggestion
    >>> G = nx.Graph()  # Create a graph with 5 nodes and 3 connected components
    >>> G.add_nodes_from(range(5))
    >>> G.add_edges_from([(0, 2), (3, 4)])
```

Reply:
true, thanks!!